index.html

```diff
@@ -56,7 +56,74 @@
           width: 100,
           height: 65,
           id: 'cta-logo',
-        }]
+        }],
+        refNote: "Since the WHATWG publishes living standards, each WHATWG standard listed below references the earliest commit in 2019 instead of the living standard. Devices must support the specified version of each WHATWG standard or later.",
```

Comment:
```suggestion
        refNote: "Since the WHATWG publishes living standards, each WHATWG standard listed below references the earliest commit in 2019 instead of the living standard. Devices must support the specified snapshot version of each WHATWG standard or later.",
```

Reply:
good point, thanks. Updated.